sdk/core/azure-core-management/src/main/java/com/azure/core/management/CreatedByType.java

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.management;
+
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import java.util.Collection;
+
+/** Defines values for CreatedByType. */
+public final class CreatedByType extends ExpandableStringEnum<CreatedByType> {
```

Comment:
Class name derived from specs, though we should be able to change to a better name if applicable.